commercetools-models/src/test/java/io/sphere/sdk/subscriptions/commands/AbstractQueueIntegrationTest.java

```diff
@@ -0,0 +1,43 @@
+package io.sphere.sdk.subscriptions.commands;
+
+import com.microsoft.azure.servicebus.*;
+import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import io.sphere.sdk.test.IntegrationTest;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+
+import java.time.Duration;
+import java.util.concurrent.CompletableFuture;
+
+import static io.sphere.sdk.subscriptions.SubscriptionFixtures.*;
+
+
+public abstract class AbstractQueueIntegrationTest extends IntegrationTest {
+
+
+    @AfterClass
+    public static void consumeMassages() throws Exception {
+        assumeHasAzureSBEnv();
+        final String connectionString = azureSBConnectionStringFromEnv();
+        final QueueClient queueClient = new QueueClient(new ConnectionStringBuilder(connectionString), ReceiveMode.PeekLock);
+        receiveMessages(queueClient);
+        Thread.sleep(60 * 1000);
```

Comment:
I think it's fine here, but may me be can retrieve the timeout via a system property or environment variable? This would make it easier to increase the timeout in our build pipeline without us having to change the code.